app/assets/javascripts/components/escape-link.js

```diff
@@ -0,0 +1,49 @@
+/* global ga:readonly */
+
+function EscapeLink ($module) {
+  this.$module = $module
```

Comment:
Are you purposley leaving out semi colons at the end of lines @markheathy?

Reply:
No, its a GDS file. I just bumped the exit button version.